webapp/channels/src/components/sidebar/sidebar_header/sidebar_header.scss

```diff
@@ -13,6 +13,7 @@
         font-family: Metropolis, sans-serif;
         font-size: 16px;
         gap: 1px;
+        color: var(--sidebar-header-text-color);
```

Comment:
Thanks @svelle. I think we may actually want to make the change on the button component (_buttons.scss) and use the `--sidebar-text` instead. I'll commit a change for that

Reply:
@matthewbirtch perfect, thank you. should I revert the changes to the sidebar_header.scss then?